src/models/GitIndex.js

```diff
@@ -116,8 +117,8 @@ export class GitIndex {
    */
   constructor (index /*: any */) {
     this._dirty = false
-    if (Buffer.isBuffer(index)) {
-      this._entries = parseBuffer(index)
+    if (isBuffer(index)) {
+      this._entries = parseBuffer(Buffer(index))
```

Comment:
This can be just `this._entries = parseBuffer(index)`. We just confirmed it is a buffer, no need to re-create it. If we did need to create a buffer from say a string, the prefered way not is `Buffer.from(index)` since `Buffer(index)` got deprecated and the standard linter won't allow it.

Reply:
If you do this then it fails the checksum. Not sure why. :/

Reply:
Hmm, that's suspect. I'm thinking there may be an easier way out of this mess. With rollup I needed to specify a buffer module for the browser, but now that the build system uses Webpack it may not be needed anymore. I'm tempted to try just removing all the `import { Buffer } from 'buffer'`lines and see if that fixes this problem.

Reply:
Definitely give it a shot! It is clearly a superficial difference between Buffer packages, and any way that results in a single package being used would surely resolve it.